parser/src/test/scala/hmda/parser/fi/lar/LarGenerators.scala

```diff
@@ -91,8 +91,7 @@ trait LarGenerators extends FIGenerators {
       "30", "31", "32", "33", "34", "35", "36", "37", "38", "39",
       "40", "41", "42", "44", "45", "46", "47", "48", "49",
       "50", "51", "53", "54", "55", "56",
-      "60", "66", "69",
-      "72", "74", "78"
```

Comment:
@schbetsy I've updated the comment to reflect what @lesgou said in hers. The list comes from Census, but only states, DC and PR need to file. I would assume that is written somewhere in the HDMA rule.